modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/linear_friction.i

```diff
@@ -0,0 +1,101 @@
+mu = 1.1
+rho = 1
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
+    dim = 2
+    dx = '20'
+    dy = '1.0'
+    ix = '10'
+    iy = '4'
+  []
+[]
+
+[Modules]
+  [NavierStokesFV]
+    compressibility = 'incompressible'
+    porous_medium_treatment = true
+
+    density = 'rho'
+    dynamic_viscosity = 'mu'
+    porosity = 'porosity'
+
+    initial_velocity = '1 1e-6 0'
```

Comment:
We probably don't need these small perturbations from a 0 initial condition. We may have needed them at some point, but I hope we don't still